src/world_model/world_model.py

```diff
@@ -0,0 +1,113 @@
+import numpy as np
+from numpy import random
+from numpy.random import choice
+from gymnasium import Env, ResetNeeded
+
+# TODO: add typing
```

Comment:
Please do, I'm a type-driven creature and it's pretty hard for me to read code without types